crates/deno_task_shell/src/parser.rs

```diff
@@ -261,6 +278,8 @@ pub enum WordPart {
   Command(SequentialList),
   /// Quoted string (ex. `"hello"` or `'test'`)
   Quoted(Vec<WordPart>),
+  /// Tilde prefix (ex. `~user/path`)
```

Comment:
Is this representing both standalone `~` for `$HOME` as well as `~user`? If so, I would document it:
```suggestion
  /// Tilde prefix (ex. `~user/path` or `~/bin`)
```

Reply:
Yes!